app/code/community/Algolia/Algoliasearch/Helper/Algoliahelper.php

```diff
@@ -84,6 +84,16 @@ public function mergeSettings($index_name, $settings)
         } catch (\Exception $e) {
         }
 
+        if (isset($settings['attributesToIndex'])) {
+            $settings['searchableAttributes'] = $settings['attributesToIndex'];
+            unset($settings['attributesToIndex']);
+        }
+
+        if (isset($onlineSettings['attributesToIndex'])) {
+            $onlineSettings['searchableAttributes'] = $onlineSettings['attributesToIndex'];
+            unset($onlineSettings['attributesToIndex']);
+        }
+
         $removes = array('slaves');
```

Comment:
Yes, let's do it now. Next release we gonna have events there and it'll cause a BC break. 
Good catch, thanks.